partitioned-table.md

```diff
@@ -163,6 +163,192 @@ Range partitioning is particularly useful when one or more of the following cond
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE employees (
+    id INT NOT NULL,
+    hired DATE NOT NULL DEFAULT '1970-01-01',
+    store_id INT
+)
+PARTITION BY LIST (store_id) (
+    PARTITION pNorth VALUES IN (1, 2, 3, 4, 5),
+    PARTITION pEast VALUES IN (6, 7, 8, 9, 10),
+    PARTITION pWest VALUES IN (11, 12, 13, 14, 15),
+    PARTITION pCentral VALUES IN (16, 17, 18, 19, 20)
+);
+```
+
+After creating the partitions as above, you can easily add or delete records related to a specific region in the table. For example, suppose that all stores in the East region (East) are sold to another company. Then all the row data related to the store employees of this region can be deleted by executing `ALTER TABLE employees TRUNCATE PARTITION pEast`, which is much more efficient than the equivalent statement `DELETE FROM employees WHERE store_id IN (6, 7, 8, 9, 10)`.
+
+You can also execute `ALTER TABLE employees DROP PARTITION pEast` to delete all related rows, but this statement also deletes the `pEast` partition from the table definition. In this situation, you must execute the `ALTER TABLE ... ADD PARTITION` statement to recover the original partitioning scheme of the table.
+
+Unlike Range partitioning, List partitioning does not have a similar `MAXVALUE` partition to store all values that do not belong to other partitions. Instead, all expected values of the partition expression must be included in the `PARTITION ... VALUES IN (...)` clause. If the value to be inserted in an `INSERT` statement does not match the column value set of any partition, the statement fails to execute and an error is reported. See the following example:
+
+```sql
+test> CREATE TABLE t (
+    ->   a INT,
+    ->   b INT
+    -> )
+    -> PARTITION BY LIST (a) (
+    ->   PARTITION p0 VALUES IN (1, 2, 3),
+    ->   PARTITION p1 VALUES IN (4, 5, 6)
+    -> );
+Query OK, 0 rows affected (0.11 sec)
+
+test> INSERT INTO t VALUES (7, 7);
+ERROR 1525 (HY000): Table has no partition for value 7
+```
+
+To ignore the error type above, you can use the `IGNORE` keyword. After using this keyword, if a row contains values that do not match the column value set of any partition, this row will not be inserted. Instead, any row with matched values is inserted, and no error is reported:
+
+```sql
+test> TRUNCATE t;
+Query OK, 1 row affected (0.00 sec)
+
+test> INSERT IGNORE INTO t VALUES (1, 1), (7, 7), (8, 8), (3, 3), (5, 5);
+Query OK, 3 rows affected, 2 warnings (0.01 sec)
+Records: 5  Duplicates: 2  Warnings: 2
+
+test> select * from t;
++------+------+
+| a    | b    |
++------+------+
+|    5 |    5 |
+|    1 |    1 |
+|    3 |    3 |
++------+------+
+3 rows in set (0.01 sec)
+```
+
+### List COLUMNS partitioning
+
+> **Warning:**
+>
+> List COLUMNS partitioning is an experimental feature. It is not recommended that you use it in the production environment.
```

Comment:
```suggestion
> List COLUMNS partitioning is an experimental feature since v5.0 GA. It is not recommended that you use it in the production environment.
```